.github/workflows/build-and-deploy.yml

```diff
@@ -6,8 +6,6 @@ on:
    branches:
     - main
   pull_request:
-    branches:
-    - main
```

Comment:
Is this to make PRs with a non-main-branch target (like this one) build?

Reply:
yes